packages/core/src/textures/GLTexture.ts

```diff
@@ -66,13 +66,13 @@ export class GLTexture
          * Type copied from baseTexture
          * @member {number}
          */
-        this.type = 6408;
+        this.type = 5121;
```

Comment:
Let's use `import('@pixi/constants').TYPES.UNSIGNED_BYTE` instead of a number literal.